src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -1415,8 +1415,6 @@ def load_arguments(self, _):
                 'For more information on "Auto" mode see aka.ms/aks/nap.'
             )
         )
-        # In update scenario, use emtpy str as default.
-        c.argument('ssh_access', arg_type=get_enum_type(ssh_accesses), is_preview=True)
```

Comment:
Please leave a history note and mark it as a breaking change. Also release a new major version.

Reply:
updated release note and bumped a new version.